unicodetools/org/unicode/text/utility/Settings.java

```diff
@@ -17,7 +17,7 @@
      * Used for the default version.
      */
     public static final String latestVersion = "14.0.0";
-    public static final String lastVersion = "13.0.0"; // last released version
+    public static final String lastVersion = "13.1.0"; // last released version
```

Comment:
FYI  I just hope that this does not throw off the generators for Unicode 14 data...

Reply:
```suggestion
    public static final String lastVersion = "13.1.0"; // last released version
    // TODO: there is no Unicode 13.1, see https://github.com/unicode-org/unicodetools/issues/100
```

Reply:
suggested change lg